tools/debug/README.md

```diff
@@ -0,0 +1,37 @@
+# DebugSymbolViewer
+
+The Debug Symbol viewer will help the developer when a crash occurs.  
+It will show some call stack information when the application has faced an abort. To be precise, it will show the symbols corresponding to the stack addresses present in the kernel text address range.
```

Comment:
Please mention the name and meaning of each column that we will print. 
Also, you need to highlight why we print dump address and symbol address separately. 
Dump address corresponds to an address on the stack. It might be an address of any line in the middle of a function, which was getting executed at the time of dump. 
Symbol address is the address of the first instruction of a function. Since our map file only provides symbol address, we use this to print the symbol name and line number. 
Hence, the line number printed here is the line number of the first line of the function. However, the execution might be at a later line at the time of crash. The user needs to manually check this during debugging.